### ISSUE TITLE
Rule proposition: max-attributes-per-line with "ignores" option

### ISSUE THREAD

**Issue by maico910** [vuejs/eslint-plugin-vue#1360]
**What rule do you want to change?**
`vue/max-attributes-per-line`

**Does this change cause the rule to produce more or fewer warnings?**
No

**How will the change be implemented? (New option, new default behavior, etc.)?**
a way to ignore some tags, like `vue/singleline-html-element-content-newline`'s option: `ignores` , is usefull working with framework's tag

**Please provide some example code that this change will affect:**

```vue
"vue/max-attributes-per-line": ["error", {
  "singleline": 1,
  "multiline": {
    "max": 1,
    "allowFirstLine": false,
    "ignores": ["q-item-label"] // example for Quasar framework's tag
  }
}],
```

**What does the rule currently do for this code?**
Check attributes per line for **every tag**

**What will the rule do after it's changed?**
Possibility to personalize to ignore some tags, creating more readable code in some cases

**Additional context**

None

**Comment by ota-meshi** [comment]
Thank you for this issue.

I don't know how that option works. Could you please provide some examples, including edge cases?
Also, could you explain to me why it is readable?

**Comment by ota-meshi** [comment]
I couldn't understand this option. So I close this issue.

**Comment by maico910** [comment]
Sorry.... I was busy and I forgot to answer...

Now I'm using Vue's framework Quasar, I have this element with it's custom props:

![image](https://user-images.githubusercontent.com/24510745/102519566-6979bc80-4071-11eb-91f0-db4304ca98bb.png)

I have two codes, without and with `max-attributes-per-line` rule:
![image](https://user-images.githubusercontent.com/24510745/102519223-f96b3680-4070-11eb-83c9-02ce1200ffaa.png)

I'am looking for some setting to make eslint to ignore some tags, it's will be good, like this:
![image](https://user-images.githubusercontent.com/24510745/102520129-223ffb80-4072-11eb-9464-216ffed150e7.png)

I don't know if this is possible or no for this rule, but I had already seen `ignores` option for another rules, like `vue/singleline-html-element-content-newline` and `vue/multiline-html-element-content-newline` :
![image](https://user-images.githubusercontent.com/24510745/102520555-97133580-4072-11eb-9392-9abc69b86277.png)

Anyway, thanks for the attention and I really apologize for the delay in the response

**Comment by MrEmelianenko** [comment]
+1 for this feature

**Comment by Eschricht** [comment]
I would also like this feature, preferably with regex support.

My usecase is that I'm using UnoCSS with the attributify preset, with a prefix.
I generally like to have one attribute per line but when adding a lot of UnoCSS attributes, the template can get a bit bulky, see the image below for tiny reference.

![image](https://github.com/vuejs/eslint-plugin-vue/assets/18286634/8de77767-8c09-4ee7-8d51-bf2ec3f25ba8)

**Comment by htycode** [comment]
+1, I would also like this feature

**Comment by htycode** [comment]
I would also like this feature

Use Case / Motivation:
Developers frequently use short, repetitive attributes (such as directives, event handlers, and binding syntax) that do not affect line clutter. However, forcing line breaks for these attributes actually reduces readability and breaks the visual consistency of simple components.

For example:
```
<!-- 'vue/max-attributes-per-line': ['error', { singleline: { max: 4 }}]-->
<!-- ❌ Error: Too many attributes-->
<component v-if="show" v-model="state.tableQueryParams.tenantId" :xs="24" :sm="12" :md="12" :lg="8" />
<!-- ❌ No error-->
<component 
    v-if="show" 
    v-model="state.tableQueryParams.tenantId" 
    :xs="24" 
    :sm="12" 
    :md="12" 
    :lg="8" 
/>

<!-- 'vue/max-attributes-per-line': ['error', { singleline: { max: 4 },ignores: ['xs', 'sm', 'md', 'lg', 'xl'] }]-->
<!-- Desired behavior (with ignores: ['xs', 'sm', 'md', 'lg', 'xl']) -->
<!-- ✅ No error: Only "v-if",v-model is counted -->
<component v-if="show" v-model="state.tableQueryParams.tenantId" :xs="24" :sm="12" :md="12" :lg="8" />
```

When configuring singleline: { max: 4 }, even concise attributes like :xs="24" :sm="12" :md="12" :lg="8" :xl="4" are counted and trigger forced line breaks—yet these short attributes do not cause line overcrowding.

With an ignores option, we can exclude these short attributes from the count. This maintains line brevity while still enforcing format restrictions for long attributes.
